src/pages/adopters.md

```diff
@@ -45,4 +45,5 @@ All organisations are sorted alphabetically by the first letter of their English
 | Vivo                      | https://www.vivo.com/         | Build a more resilient and elastic hybrid-multi-cloud platform using Karmada                                                                                                                                              |                                                                                                                                                                           |
 | Youzan                    | https://www.youzan.com/       | Manage both Stateless and Stateful on-line workloads across multiple clusters                                                                                                                                             |                                                                                                                                                                           |
 | Zhejiang Lab              | https://www.zhejianglab.com/  | Use Karmada to build multi-cluster platform                                                                                                                                                                               |                                                                                                                                                                           |
-| ZTO                       | https://www.zto.com           | Use Karmada to build multi-cluster platform                                                                                                                                                                               |                                                                                                                                                                           |
+| ZTO                       | https://www.zto.com           | Use Karmada to build multi-cluster platform                                                                                                                                                                                |                                                                                                                                                                          |
+| WPS                       | https://www.wps.cn           | Use Karmada to build a multi-cluster and multi-cloud AI platform.                                                                                                                                                                                 |                                                                                                                                                                          |
```

Comment:
```suggestion
| WPS                       | https://www.wps.cn           | Use Karmada to build a multi-cluster and multi-cloud AI platform                                                                                                                                                                                 |                                                                                                                                                                          |
```
For consistency

Reply:
no big deal. I think we can have the `.` at the end.
Sure, we can make an update for all of them. Thanks for pointing that out!